src/main/sensors/sensors.h

```diff
@@ -40,7 +40,7 @@ typedef union flightDynamicsTrims_u {
     flightDynamicsTrims_def_t values;
 } flightDynamicsTrims_t;
 
-#define CALIBRATING_GYRO_CYCLES             10000
+#define CALIBRATING_GYRO_TIME           3000000 // gyro calibration time in microseconds
```

Comment:
It's best to call it `CALIBRATING_GYRO_TIME_USEC` and omit the commentary IMO

Reply:
Good point, @DieHertz but I think the commonly used suffix for microseconds in Betaflight is `_US`.

Reply:
I'm fine with whatever is accepted :-)